### ISSUE TITLE
Make it work on non-macOS platforms

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/macbook-pro-battery-test#8]
I'd also like to run the tests on a few other OSes and hardware, just for grins, like my Lenovo T420 and it's double-capacity battery.

Most thing should work the same across most Linux distros or even something like Cygwin or Cmder (though not WSL under Windows, because it's not a _full_ Linux instance), but I think the `pmset` command would need to be behind a flag.

**Comment by oxyc** [comment]
At least on Arch I use [`/sys/class/power_supply/AC/online`](https://github.com/oxyc/dotfiles/blob/master/.local/bin/battery) to check if it's on AC or battery.

Something like

```sh
case "$OSTYPE" in
  darwin*) [[ $(pmset -g ps | head -1) =~ "AC Power" ]] && AC=1 || AC=0 ;;
  linux*) AC=$(cat /sys/class/power_supply/AC/online) ;;
esac

if [ $AC -eq 1 ]; then
  printf "\033[0;31mPlease unplug power before starting the test.\033[0m\n"; exit 1;
fi
```

**Comment by geerlingguy** [comment]
I merged PR #9 and will try to test on Fedora later today.
